protos/shell/shell.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package mavsdk.rpc.shell;
+
+option java_package = "io.mavsdk.shell";
+option java_outer_classname = "ShellProto";
+
+/**
+ * Shell Service allow to communicate with vehicle's system shell.
+ */
+service ShellService {
+    // Communicate with a vehicle's Shell.
+    rpc SetShellMessage(SetShellMessageRequest) returns(SetShellMessageResponse) {}
```

Comment:
Just thinking about the wording. It will be called with something like `drone.shell.set_shell_message()`. Could we do something like `rpc Send` (which will become `drone.shell.send()`) or `rpc Write (which will become `drone.shell.write()`)? 

For instance, we do `drone.camera.set_mode()` and not `drone.camera.set_camera_mode()`.

@julianoes @irsdkv: do you have an opinion?

Reply:
I think that this is more convenient naming. If with this names will be possible to produce wrappers (to Python for example) so this will great I think.

Reply:
I'm favor of using the namespace and not repeating the words.

Reply:
> I  think that this is more convenient naming.

Not sure I understand that. So you mean that `SetShellMessage` is more convenient than `Send`?

Right now you write:

```python
shell_message = ShellMessage(...)
drone.shell.set_shell_message(shell_message)
```

Which to me sounds a bit confusing, because `set_` is a "setter". Here, it is not a setter: it actually sends a message to the drone. If you call it `rpc Send(SendMessageRequest) returns(SendMessageResponse) {}`, then the generated code becomes:

```python
shell_message = ShellMessage(...)
drone.shell.send(shell_message)
```

Which looks clearer to me.

Reply:
> Not sure I understand that. 

Oh sorry it is my mistake. 
I mean that `Send` more convenient than `SetShellMessage` of course.

Reply:
Ok! Let's change it to `Send` and merge that, then :-)

Reply:
Please check is I understand necessary changes in naming correctly (in case of `Request`/`Response` names)

Reply:
Looks like `MAVSDK-Python` generator dont understand me it :)
```python
        request = shell_pb2.SendRequest()   # Need to be SendMessageRequest
        shell_message.translate_to_rpc(request.shell_message)
```
Will change to `SendRequest`/`SendResponse`

Reply:
> Looks like MAVSDK-Python generator dont understand SendMessageRequest

Yes, I realize I made a typo above. It expects the same name everywhere: <name>, <name>Request, <name>Response, so:

This is valid:

```
rpc Send(SendRequest) returns(SendResponse) {}
```

This is not valid:

```
rpc Send(SomethingRequest) returns(SomethingElseResponse) {}
```